cmd/lookout/event.go

```diff
@@ -46,6 +46,7 @@ func (c *EventCommand) openRepository() error {
 }
 
 func (c *EventCommand) resolveRefs() (*lookout.ReferencePointer, *lookout.ReferencePointer, error) {
+	log.Infof("Resolving to/from references")
```

Comment:
should this be not capitalized?

Reply:
seems to be fine, as this is a line-oriented logging, no?

Reply:
both are fine to me, I was proposing lowercase for consistency with the other ones